.rubocop.yml

```diff
@@ -79,3 +79,29 @@ Style/ParenthesesAroundCondition:
 
 Style/PreferredHashMethods:
   Enabled: false
+
+Minitest/AssertEmptyLiteral:
+  Enabled: true
+
+Minitest/AssertNil:
+  Exclude:
+    - 'test/graphql/queries/personal_access_token_query_test.rb'
+  Enabled: true
+
+Minitest/RefuteNil:
+  Enabled: true
+
+Layout/ArgumentAlignment:
+  EnforcedStyle: with_fixed_indentation
```

Comment:
I'd vote for `with_first_argument` (default) here. It would get rid of so much changes as well as would force the argument alignment to be more readable. But at the same time this should allow us to write arguments on a separate lane in case the default indentation goes to far right. I.e. it should also allow:
```ruby
method_call(
  first_long_arg,
  second_longer_arg
)
```
I mean, `with_fixed_indentation` should allow that as well, but it leads to a lot of changes here.